specs/altair/p2p-interface.md

```diff
@@ -182,6 +182,8 @@ Request and Response remain unchanged. A `ForkDigest`-context is used to select
 
 Per `context = compute_fork_digest(fork_version, genesis_validators_root)`:
 
+[0]: # (eth2spec: skip)
```

Comment:
example of using "comments" in markdown to specify parsing directives, in this case, to skip the pyspec parsing of the next markdown element (the table)

convention inspired by this: https://stackoverflow.com/a/32190021